lively.freezer/src/bundler.js

```diff
@@ -480,22 +480,39 @@ export default class LivelyRollup {
       }
     }
 
+    let absolutePath;
+
     if (id.startsWith('.')) { // handle some kind of relative import
       try {
-        return this.resolved[resolutionId(id, importer)] = await this.resolveRelativeImport(importer, id);
+        absolutePath = await this.resolveRelativeImport(importer, id);
+        if (this.belongsToExcludedPackage(absolutePath)) return null;
+        return this.resolved[resolutionId(id, importer)] = absolutePath;
       } catch (err) {
         return null;
       }
     }
 
     if (!id.endsWith('.js') && !isCdnImport) {
```

Comment:
Are we sure this works? Above, I could only find a function with this name and I do not see a variable declared anywhere in this method? :thinking: